# Oblig3
Oblig 3 i software engineering og testing


Først lagde jeg repository. 

Jeg la til prosjektfila inn i github. 

Så gikk jeg på inn på actions og la til en workflow i prosjektet mitt som het python package. 

Og så  comitta jeg filene på nytt og da kjørte testene og fikk feil på alle med exit code 2. 

 

Det jeg måtte gjøre var at siden “test” modulen lå inne i selve prosjekt mappa så ville ikke testene kjøre riktig. Så jeg tok de ut av mappa og pusha de til github for seg selv. 

 

Så pushet jeg prosjekt mappa inn på github på nytt og fikk riktig på alle testene. 

![image](https://user-images.githubusercontent.com/105790048/198331613-26303175-e072-44ae-b3a0-27df6c8862ca.png)
